icu4c/source/test/testdata/rbbitst.txt

```diff
@@ -1981,6 +1981,57 @@ Bangkok)•</data>
 #これらの連絡先はデバイスをロック解除しなくても表示され -> これらの▁連絡先は▁デバイスを▁ロック▁解除しなくても▁表示され
 <data>•\u3053\u308C\u3089\u306E•\u9023\u7D61\u5148\u306F•\u30C7\u30D0\u30A4\u30B9\u3092•\u30ED\u30C3\u30AF•\u89E3\u9664\u3057\u306A\u304F\u3066\u3082•\u8868\u793A\u3055\u308C•</data>
 
+# Test the differences in ko with or without lw=phrase.
```

Comment:
There's a second copy of rbbitst.txt in ICU4J.
The two are supposed to always be identical; changes in one that aren't in the other are at risk of being overwritten by other updates.

Reply:
Thank you, Andy !   Should I fold the corresponding Java (perhaps 1 line code change) into this PR?  Oh. You mentioned it below. I'll do that and update the PR.